lib/build-database.js

```diff
@@ -10,7 +10,16 @@ const memoize = require( 'memoizee' )
 const { reduce, map, forEach } = require( 'fast.js' )
 const { unload } = require( 'freshy' )
 const { omit } = require( 'lodash' )
-const { firstLetters, toEnglish, toHindi, toShahmukhi, toUnicode } = require( 'gurmukhi-utils' )
+const {
+  firstLetters,
+  stripEndings,
+  stripVishraams,
+  stripAccentts,
```

Comment:
Two t in accents

Reply:
And I don't see where it's used anyways

Reply:
very bad very bad, -1000 for the sloppiness